Removes an unsupported Podman section and adds a link to OMR (#44040)
openshift/openshift-docs: modules/ipi-install-creating-a-disconnected-registry.adoc

:_content-type: PROCEDURE
[id="ipi-install-creating-a-disconnected-registry_{context}"]

> = Creating a disconnected registry on a registry node

Reviewer: I'm not sure if we want to use `mirror registry` instead of `disconnected registry` in this section. 
Mirror registry is introduced in 
https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-creating-registry.html and it sounds like more reasonable in the installation usage. `disconnected registry` sounds like the registry is located in a disconnected network, but it should have access to Quay at least. 
BTW, I see we're also saying mirror registry in the following steps.
```
This procedure explains how to install the mirror registry for Red Hat OpenShift on a local host
```

Author: @gpei are you referring to the terms (swapping the term 'mirror registry' for 'disconnected registry'), or are you talking about the tool ("Mirror Registry for Red Hat OpenShift") itself? Including the procedure for "Mirror Registry for Red Hat OpenShift" was encouraged by Eric Rich. 

Please note that I cannot swap 'mirror registry' for 'disconnected registry' if that's what you're suggesting. The terms "Mirror Registry for OpenShift" is for the actual tool itself; that can't be renamed because of Marketing. It's a confusing name with no flexibility. I tried to get it changed.

> = Creating a disconnected registry on a registry node

Reviewer: I'm not sure if we want to use `mirror registry` instead of `disconnected registry` in this section. 
Mirror registry is introduced in 
https://docs.openshift.com/container-platform/4.10/installing/disconnected_install/installing-mirroring-creating-registry.html and it sounds like more reasonable in the installation usage. `disconnected registry` sounds like the registry is located in a disconnected network, but it should have access to Quay at least. 
BTW, I see we're also saying mirror registry in the following steps.
```
This procedure explains how to install the mirror registry for Red Hat OpenShift on a local host
```

====

.Procedure

. Download the `mirror-registry.tar.gz` package for the latest version of the _mirror registry for Red Hat OpenShift_ found on the link:https://console.redhat.com/openshift/downloads#tool-mirror-registry[OpenShift console *Downloads*] page.

. Install the _mirror registry for Red Hat OpenShift_ on your local host with your current user account by using the `mirror-registry` tool. For a full list of available flags, see "mirror registry for Red Hat OpenShift flags".
+
[source,terminal]
----
$ sudo ./mirror-registry install \
  --quayHostname <host_example_com> \
  --quayRoot <example_directory_name>
----

> . Use the user name and password generated during installation to log into the registry by running the following command:

Reviewer 2: ```suggestion
. Use the user name and password that was generated during installation to log into the registry by running the following command:
```

+
[source,terminal]
----
$ podman login --authfile pull-secret.txt \
  -u init \
  -p <password> \
  <host_example_com>:8443> \
  --tls-verify=false <1>
----
<1> You can avoid running `--tls-verify=false` by configuring your system to trust the generated rootCA certificates. See "Using SSL to protect connections to Red Hat Quay" and "Configuring the system to trust the certificate authority" for more information.
+
[NOTE]
====
You can also log in by accessing the UI at `\https://<host.example.com>:8443` after installation.
====

> . You can mirror {product-title} images after logging in. Depending on your needs, see either the "Mirroring the {product-title} image repository" or the "Mirroring Operator catalogs for use with disconnected clusters" sections of this document.

Reviewer 2: ```suggestion
. You can mirror {product-title} images after logging in. Depending on your requirements, see either the "Mirroring the {product-title} image repository" or the "Mirroring Operator catalogs for use with disconnected clusters" sections that follow.
```

+
[NOTE]
====

> If there are issues with images stored by the _mirror registry for Red Hat OpenShift_ due to storage layer problems, you can remirror the {product-title} images, or reinstall mirror registry on more stable storage.

Reviewer 2: > Avoid using expletive constructions such as it is, there are, and there is, which hide the subject of the sentence.

If you know what kind of issues, it would be better to say that. Like, is this here because of errors during installation?

```suggestion
If there are issues with images stored by the _mirror registry for Red Hat OpenShift_ due to storage layer problems, you can remirror the {product-title} images or reinstall mirror registry on more stable storage.
```
